src/core/keyvault_access_policy.tf

```diff
@@ -29,6 +33,18 @@ resource "azurerm_key_vault_access_policy" "adgroup_admin_common" {
   certificate_permissions = ["Get", "List", "Update", "Create", "Import", "Delete", "Restore", "Recover", ]
 }
 
+# kv-common reader policy
+resource "azurerm_key_vault_access_policy" "adgroup_directory_readers" {
```

Comment:
directory-readers group will contains all managed identity that needs to read Entra ID UUIDs.
Managed identity can be cross project/platform/subscription so is not a good idea give this kv read access policy to this group.

An alternative is to create a directory-readers group scoped to subscription (something like io-p-directory-readers), but again I'm not sure to assign the kv reader policy to entire scoped group is a good idea.

Reply:
Discussed on Slack, moving to create an access policy for each identity